tests/utils.test.ts

```diff
@@ -2,6 +2,8 @@ import { expect, test } from "@playwright/test";
 
 import { runProcess, convertDateToTimeStampS } from "../src/utils";
 
+import mapProjects from "../../mandates-map";
```

Comment:
How come this change?

Reply:
I forgot exactly how this happened, I think I was messing around with another branch and accidentally committed this. I'm removing this to match the main branch!